docs/docsite/rst/porting_guides/porting_guide_2.0.rst

```diff
@@ -149,7 +149,7 @@ Should now be::
         a: 1
 
 * Setting any_errors_fatal on a task is no longer supported. This should be set at the play level only.
-* Bare variables in the `environment` dictionary (for plays/tasks/etc.) are no longer supported. Variables specified there should use the full variable syntax: '{{foo}}'.
+* Bare variables in the `environment` dictionary (for plays/tasks/and so on) are no longer supported. Variables specified there should use the full variable syntax: '{{foo}}'.
```

Comment:
```suggestion
* Bare variables in the `environment` dictionary (for plays, tasks, and so on) are no longer supported. Variables specified there should use the full variable syntax: '{{foo}}'.
```

Reply:
Thanks for the suggestion, it reads even better.